llvm-external-projects/iree-compiler-api/lib/CAPI/Compiler.cpp

```diff
@@ -9,11 +9,16 @@
 #include "iree/compiler/ConstEval/Passes.h"
 #include "iree/compiler/Dialect/VM/IR/VMOps.h"
 #include "iree/compiler/Dialect/VM/Target/Bytecode/BytecodeModuleTarget.h"
+#include "iree/compiler/Dialect/VM/Target/init_targets.h"
```

Comment:
If registerVMTargets is not needed, then this can be dropped (and will fix a build issue).